source/fundamentals/builders/aggregates.txt

```diff
@@ -110,6 +107,34 @@ into a new field called ``rating``, effectively renaming the field.
    :dedent:
 
 
+Documents
+---------
+
+Use the ``documents()`` method to create a
+:manual:`$documents </reference/operator/aggregation/documents/>`
+pipeline stage that returns literal documents from input values.
+
+.. important::
+
+   If you use a ``$documents`` stage in an aggregation pipeline, it must be the first 
+   stage in the pipeline.
```

Comment:
This is based on a sentence in [this Google doc](https://docs.google.com/document/d/1yRcAcMUJysh4B-OMUDp2XsjgPgozYj7Mo2PwfepTC_U/edit#). Did I interpret it correctly?

Reply:
This is correct.

The aggregation must also run on the database, rather than on a specific collection. This is called the [aggregation level](https://github.com/mongodb/mongo-java-driver/commit/dee0358bff6bae9b65371a0967180bd342260245#diff-cb241c9477e484cb6edcd5e8e0df80078e9a94f813167089bdc5c7f1d79e1dd7R289), but practically it means that one must use:

            mongoClient.getDatabase("dbName").aggregate(...);

rather than:

            mongoClient.getDatabase("dbName").getCollection("cName").aggregate(...);